src/js/components/DataChart/stories/Everything.js

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+
+import { Box, DataChart, Grommet } from 'grommet';
+import { grommet } from 'grommet/themes';
+
+const data = [];
+for (let i = 0; i < 13; i += 1) {
+  const v = -Math.sin(i / 2.0);
+  const v2 = Math.cos(i / 2.0);
+  data.push({
+    date: `2020-07-${((i % 30) + 1).toString().padStart(2, 0)}`,
+    amount: Math.floor(v * 100),
+    need: Math.floor(v2 * 10),
+  });
+}
+
+const Example = () => (
+  <Grommet theme={grommet}>
+    <Box align="center" justify="start" pad="large">
+      <DataChart
+        data={data}
+        property={[
+          'date',
```

Comment:
It's unclear to me why `date` needs to be explicitly mentioned here?

Reply:
The current API requires that any data property you want to include needs to be specified in `property`. In this case, `axis` uses it. It does allow you to do: `<DataChart data={data} property={['date', 'amount']} />` and it will default everything. But, perhaps that's making things too easy?

Reply:
revised how this works